src/Components/ArrayQuestions/ArrayMethodsS.scss

```diff
@@ -1,37 +1,35 @@
 @import "../../Sass/main.scss";
 
 .ArrayMethods {
-    display: flex;
-    align-items: center;
-    flex-direction: column;
-
-    .question {
+    .ArrayMethods-card.card {
         display: flex;
         flex-direction: column;
         align-items: center;
-    }
+        width: 60rem;
+        height: 20rem;
```

Comment:
Typically speaking it can be tricky to set your width's and height's unless you are sure that you want them all to be the same size. Might make sense to let them grow and shrink on their own in the future